alert-policies/ansible-automation-controller/LowInstanceMemory.yml

```diff
@@ -0,0 +1,37 @@
+name: Low Instance Memory
+
+description: |+
+  This alert is triggered when the low instance memory is higher than 1024 MB during 5 minutes.
```

Comment:
```suggestion
  This alert is triggered when the low instance memory is higher than 1024 MB within 5 minutes.
```

Reply:
Hi @**paperclypse** ,
Reviewed text correction has been changed. Kindly review it.